pytest.ini

```diff
@@ -4,8 +4,9 @@
 markers =
     integration: Mark a test as an integration test.  These generally take > 30 seconds to complete.
     smoke: Mark a test as an smoke test. These often exercise the full stack
-    non_default_llm_provider: tests that exercise llm providers other than our default Open AI
-    anthropic: tests that specifically use anthropic as the llm provider
+    non_default_llm_provider: Tests that exercise llm providers other than our default Open AI
+    anthropic: Tests that specifically use anthropic as the llm provider
+    needs_server: Tests that need a server running in order to complete successfully
```

Comment:
Add the needs_server to the list of pytest markers.  Other lines just add a capitalization.